src/libraries/System.Linq.Expressions/tests/Dynamic/InvokeMemberBindingTests.cs

```diff
@@ -380,8 +375,7 @@ public static IEnumerable<object[]> SameNameObjectPairs()
             return testObjects.SelectMany(i => testObjects.Select(j => new[] { i, j }));
         }
 
-        // We're not testing compilation, but we do need Reflection.Emit for the test
-        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsNotLinqExpressionsBuiltWithIsInterpretingOnly))]
+        [Theory]
```

Comment:
Also don't see anything ref.emitting. Was probably some .NET Native issue related to dynamic.